### ISSUE TITLE
instructions need to be updated 

### ISSUE THREAD

**Issue by chrplr** [neurospin/unicog#16]
$ source /i2bm/local/Ubuntu-14.04-x86_64/brainvisa/bin/bv_env.sh
bash: /i2bm/local/Ubuntu-14.04-x86_64/brainvisa/bin/bv_env.sh: No such file or directory

**Comment by isadenghien** [comment]
The access to Anatomist and BrainVisa tools have recently changed.
The doc is up to date to 
https://www.neurospin-wiki.org/pmwiki/Main/BrainvisaSetup

**Comment by chrplr** [comment]
Merci!

Je cherchais dans /i2bm/local/brainvisa, pas dans /i2bm/brainvisa que je
n'avais pas vu bien sûr.

Mais il faut quand même updater le readme du viewer.

Je risque te t'appeler à l'aide pour faire une fusion 3D sous anatomist :-(
Tu es à neurospin aujourd'hui ?

**Comment by isadenghien** [comment]
oui je suis à NS, mais je ne pas utiliser ma workstation pour travailler.

2.44.99


Maintenant sur le réseau, il faut avoir Singularity pour lancer Anatomist et

Brainvisa ... pas sure que ca fonctionne avec la "methode" source ....


J'ai vu ça quand j'ai travailler avec Tiffany pour pyanatomist et pycrotex.


Je fais le ligthning talk la semaine prochaine ... je parlerai surment des projections

de map avec pyanatomist/pycortex/pysurfer.






________________________________
De : Christophe Pallier ***@***.***>
Envoyé : vendredi 11 juin 2021 11:42:26
À : neurospin/unicog
Cc : DENGHIEN COURCOL Isabelle; Comment
Objet : Re: [neurospin/unicog] instructions need to be updated (#16)


Merci!

Je cherchais dans /i2bm/local/brainvisa, pas dans /i2bm/brainvisa que je
n'avais pas vu bien sûr.

Mais il faut quand même updater le readme du viewer.

Je risque te t'appeler à l'aide pour faire une fusion 3D sous anatomist :-(
Tu es à neurospin aujourd'hui ?

**Comment by denisri** [comment]
Vous utilisez encore une version Ubuntu 14 ???
Oui les versions récentes (brainvisa 5) nécessitent singularity 3, qui peut se télécharger depuis https://brainvisa.info/web/download.html
Le script `bv_env.sh` est maintentant dans le conteneur, mais n'est plus nécessaire puisqu'il se charge quand on lance le conteneur singularity, avec la commande "`bv`" qui est dans le bin/ de l'install: `bv bash`, `bv anatomist`, etc.

**Comment by chrplr** [comment]
Non, non, c'est juste la doc qui date de Ubuntu 14.Il faut mettre à jour
les instructions dans le readme.md pour qu'on puisse faire fonctionner le
script de fusion écrit par Isabelle. (Et il y a un autre soucis dans le
setup.py du module unicog)
